gym/scoreboard/api.py

```diff
@@ -54,6 +54,7 @@ def upload(training_dir, algorithm_id=None, writeup=None, api_key=None, ignore_o
         training_episode_batch=training_episode_batch_id,
         training_video=training_video_id,
         env=env_info['env_id'],
+        wrappers=env_info['wrappers'],
```

Comment:
We may add something like this in the future, but not yet. For now, I'm ok with just not worrying about the fact that people may be using wrappers. It'll be clear from people's posted code. If it becomes a problem we'll deal with it.